### PR TITLE
Fix bug and golint failures of hack/verify-api-groups.sh

### PR DESCRIPTION
**Pull request by SataQiu**
**What type of PR is this?**

/kind bug
/kind cleanup

**What this PR does / why we need it**:
Fix bug and golint failures of hack/verify-api-groups.sh

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```
/sig testing

### CODE REVIEW

File: hack/verify-api-groups.sh [hunk 1]
```diff
@@ -112,8 +112,8 @@ for expected_install_package in "${expected_install_packages[@]}"; do
 	fi
 
 	for known_version_file in "${known_version_files[@]}"; do
-		if ! grep -q "${expected_install_package}/install" ${known_version_files} ; then
-			echo "missing ${expected_install_package}/install from ${known_version_files}"
+		if ! grep -q "${expected_install_package}/install" "${known_version_file}" ; then
```
Comment (BenTheElder) [inline hack/verify-api-groups.sh]: I think if you go digging the file on line 101 does not exist and we patched this out in one of the other scripts

Comment (BenTheElder) [reply hack/verify-api-groups.sh]: @liggitt might be someone to ask :^)

Comment (SataQiu) [reply hack/verify-api-groups.sh]: @BenTheElder 
Yes, this file does not exist.
```
pkg/client/clientset_generated/internal_clientset/import_known_versions.go: No such file or directory
```
But how do we fix it? Any suggestions? @liggitt

Comment (liggitt) [reply hack/verify-api-groups.sh]: this was removed in https://github.com/kubernetes/kubernetes/pull/71136

we can drop the `clientset_generated/internal_clientset` line

Comment (SataQiu) [reply hack/verify-api-groups.sh]: Thanks a lot. @liggitt 
I have updated the code.

**Pull request merged**